### PR TITLE
Some code cleanup, minor GUI fixes

### LINKED ISSUE #2730: Builder workorder list issue
### Minecolonies version

version 0.9.72 forge 14.23.4.2738

Screenshot attached:

### LINKED ISSUE #2732: Reduce log spam
version any
```
[16:02:22] [main/INFO] [minecolonies]: Finished discovering saplings
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering ores
[16:02:22] [main/INFO] [minecolonies]: Finished discovering compostables
```

### PR DESCRIPTION
**Pull request by Nightenom**
Closes #2732
Closes #2730

### Changes proposed in this pull request:
- also fixed a scrollbar offset when generating a scrolling list

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/client/gui/WindowRequestDetail.java [hunk 1]
```diff
@@ -70,7 +70,7 @@
     /**
      * Wrap after x amount of symbols the text.
      */
-    private static final int WRAP_AFTER_X = 30;
+    //private static final int WRAP_AFTER_X = 30;
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/client/gui/WindowRequestDetail.java:73]: If we don't use these you can remove them

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count |  New  | Old  | Diff  |
| ------ |:-----:|:-----:|:----:|:-----:|
| Total  | 39561 | 14978 | 8501 | +6477 |
| Errors |  594  |  28   |  18  |  +10  |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=15372&tab=Inspection&buildTypeId=bt88)

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 39561 |  2  |  2  |  0   |
| Errors |  594  |  0  |  0  |  0   |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=15468&tab=Inspection&buildTypeId=bt88)

**Review by LDTTeam-Buildsystem — APPROVED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 39561 |  2  |  2  |  0   |
| Errors |  594  |  0  |  0  |  0   |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=15492&tab=Inspection&buildTypeId=bt88)

**Review by LDTTeam-Buildsystem — APPROVED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 39559 |  2  |  4  |  -2  |
| Errors |  593  |  0  |  1  |  -1  |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=15504&tab=Inspection&buildTypeId=bt88)

**Pull request merged**